beta/project-griffon/set-up-project-griffon.md

```diff
@@ -118,13 +122,19 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 
 ### Start a Project Griffon session
 
-After the extension has been registered, start a Project Griffon session by using the following API:
+1. [Create](https://aep-sdks.gitbook.io/docs/beta/project-griffon/using-project-griffon#1-create-a-session) a new session in the [griffon UI](https://griffon.adobe.com/) by providing a base url for your application.
+
+2. Configure your iOS/Android application to handle the griffon session deeplink obtained from the griffon.
 
 {% tabs %}
 {% tab title="Android" %}
+
+Once your app is configured to handle the deeplink. You can start the griffon session by opening the deeplink provided from the griffon UI
```

Comment:
Please reword to:
"After your app is configured to handle the deeplink, start the Griffon session by opening the deeplink from the Griffon UI."